.drone.yml

```diff
@@ -44,6 +64,25 @@ steps:
       event:
         - tag
 
+  - name: deploy-dev
+    image: quay.io/mongodb/drone-helm:v3
+    settings:
+      chart: mongodb/web-app
+      chart_version: 4.12.3
+      add_repos: [mongodb=https://10gen.github.io/helm-charts]
+      namespace: docs
+      release: docs-search-transport-dev
+      values: image.tag=git-${DRONE_COMMIT_SHA:0:7},image.repository=795250896452.dkr.ecr.us-east-1.amazonaws.com/docs/${DRONE_REPO_NAME},ingress.enabled=true,ingress.hosts[0]=docs-search-transport-dev.docs.staging.corp.mongodb.com
```

Comment:
If we're planning on using the image tag from the `publish-dev` step, I think we need to make the image.tag here conform to the same name: `git-${DRONE_COMMIT_SHA:0:7}-dev`. Same with the rename of the staging tag if we plan to add the `-staging` suffix. Otherwise, I don't think the images will be pulled in properly

Reply:
good catch! done